srcpkgs/pulseeffects-legacy/template

```diff
@@ -4,18 +4,22 @@ version=4.8.5
 revision=1
 wrksrc="pulseeffects-${version}"
 build_style=meson
-hostmakedepends="itstool pkg-config gettext"
+hostmakedepends="itstool pkgconf gettext"
 makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
  gst-plugins-bad1-devel gtkmm-devel libebur128-devel lilv-devel
- pulseaudio-devel python3-gobject-devel sratom-devel
- libsndfile-devel"
-depends="calf gsettings-desktop-schemas gst-plugins-bad1
- gst-plugins-good1 pulseaudio python3-gobject python3-scipy"
+ pulseaudio-devel sratom-devel zita-convolver-devel
+ rubberband-devel libsamplerate-devel libsndfile-devel"
+depends="calf gst-plugins-good1"
 short_desc="Sound effects for systems using PulseAudio (legacy)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/wwmm/pulseeffects"
+changelog="https://github.com/wwmm/pulseeffects/blob/v${version}/CHANGELOG.md"
```

Comment:
using master for changelog instead of a tag is probably better

Reply:
> using master for changelog instead of a tag is probably better

Why? If upstream releases a new version (tag), then changelog will contain info about this new version, though the package version will remain the same. This creates confusion for users IMO.

Reply:
changelog is not for end users, it is for maintainers and void contributors. It's useful to look in it when I am updating a package, I want to see all changes in all releases, not just the one the template currently references. Also, having a variable in there is very annoying, I want to click on the link in my terminal and not have to change the version.

Reply:
OK, I'll change it

Reply:
ping @Logarithmus? Also, please don't resolve conversations that were not resolved.

Reply:
@paper42 sorry for missing it, fixed now